screens/RewardScreen/RewardScreen.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import {
+  Text,
+  View,
+  Image,
+  StyleSheet,
+  TouchableOpacity,
+  Button
+} from "react-native";
+import { Asset } from "expo-asset";
+import * as Font from "expo-font";
+import styles from "./RewardScreenStyles";
+
+export default class RewardScreen extends Component {
+  async componentWillMount() {
+    await Asset.loadAsync([require("../../assets/images/teddynew.png")]);
+
+    await Font.loadAsync({
+      "amatic-sc": require("../../assets/fonts/amatic-sc.ttf"),
+      "Satisfy-Regular": require("../../assets/fonts/Satisfy-Regular.ttf")
+    });
+  }
+
+  render() {
+    return (
+      <View>
+        <Image
+          style={styles.img}
+          source={require("../../assets/images/teddy-no-overlay.png")}
+        />
+        <View style={{ flexDirection: "row", marginTop: 15, marginLeft: 15 }}>
+          <Image
+            style={{ marginTop: 5 }}
+            source={require("../../assets/images/loc_emoji.png")}
+          />
+          <Text style={styles.text}>HØJE TAASTRUP</Text>
+        </View>
+
+        <Text style={styles.locationText}>
```

Comment:
Maybe refactor this in a few smaller subcomponents?
Also, if you'll want to make it prettier, it'll end up growing even more.